packages/adapter-cloudflare-workers/index.js

```diff
@@ -140,6 +141,30 @@ export default function ({ config = 'wrangler.toml' } = {}) {
 			const bucket_dir = `${site.bucket}${builder.config.kit.paths.base}`;
 			builder.writeClient(bucket_dir);
 			builder.writePrerendered(bucket_dir);
+		},
+
+		async emulate() {
+			const proxy = await getBindingsProxy();
+			// TODO this also needs `context` and `caches`
+			const platform = /** @type {App.Platform} */ ({ env: proxy.bindings });
```

Comment:
```suggestion
			const platform = /** @type {App.Platform} */ ({
				env: proxy.env,
				context: proxy.ctx,
				caches: proxy.caches,
				cf: proxy.cf
			});
```

Reply:
We would need to update the docs if we expose new fields on platform: https://kit.svelte.dev/docs/adapter-cloudflare#bindings

I don't know if @Rich-Harris and @dario-piotrowicz have any opinion on whether we should call it `context` for readability vs `ctx` for consistency with Cloudflare's API

Reply:
@benmccann for more context (pun intended 😅)

Some frameworks refer to it as `ctx`, as for example qwikCity: https://github.com/BuilderIO/qwik/blob/f7c5a5576bec1c42a67f6d8e4f1eb9e07cef5ef5/packages/qwik-city/middleware/cloudflare-pages/index.ts#L89

others like Nitro/Nuxt call it `context`: https://github.com/unjs/nitro/blob/e3f555acd3f74adab57db237992815dad6d683ce/src/runtime/entries/cloudflare-pages.ts#L82

Other might just expose the `waitUntil`, like Astro: https://github.com/withastro/adapters/blob/3854812c9bd3558857bb9084668925835dcf4ee0/packages/cloudflare/src/entrypoints/server.directory.ts#L44

So I think the ecosystem's a bit mixed on this... I think you should go with whatever you feel more natural for your svelteKit 🙂 

(but my personal option would be to stick with `ctx` is possible 🙂)

(PS: as far as I know it is not referred as `ctx` officially by Cloudflare either, but we do include is as such in various templates and in `getPlatformProxy` so it's sort of half-official I'd say)

Reply:
It might be beneficial to keep it as `context` just because that's what the adapters have named it already. Maybe down the line we can rename it if we need to.

https://github.com/sveltejs/kit/blob/41bb2c0fc752843651e00de133da618884c2c5ee/packages/adapter-cloudflare/src/worker.js#L12

https://github.com/sveltejs/kit/blob/41bb2c0fc752843651e00de133da618884c2c5ee/packages/adapter-cloudflare/src/worker.js#L58

Reply:
since the docs are already missing in `main`, we can handle the additional docs in https://github.com/sveltejs/kit/pull/11731#issuecomment-1909412676